### ISSUE TITLE
Regression in if-then-else docs

### ISSUE THREAD

**Issue by amagee** [jqlang/jq#2139]
The docs at https://stedolan.github.io/jq/manual/ currently say:

> Checking for false or null is a simpler notion of "truthiness" than is found in Javascript or Python, but it means that you'll sometimes have to be more explicit about the condition you want. You can't test whether, e.g. a string is empty using `if .name then A else B end`, you'll need something more like `if .name then A else B end instead`.

Note the two expressions there are identical.

Seems to be a regression introduced in https://github.com/stedolan/jq/commit/8a02b2c9c191015d8f1e5e27404e2682d76b6a91

**Comment by jeff-smith-pcln** [comment]
I noticed this as well.

I would _**also**_ offer that the "else" is **not** optional (as documented), but is (now) **required**, as shown in the following code examples (for jq 1.6):
```
> print '{"labels":{"owner1":"dls","archived":"","owner2":"data-services"}}' | jq -r '.labels? | length>0 '                                        
true

> print '{"labels":{"owner1":"dls","archived":"","owner2":"data-services"}}' | jq -r '.labels? | if length>0 then "present" else "not present" end'
present

> print '{"labels":{"owner1":"dls","archived":"","owner2":"data-services"}}' | jq -r '.labels? | if length>0 then "present" end'                   
jq: error: syntax error, unexpected end (Unix shell quoting issues?) at <top-level>, line 1:
.labels? | if length>0 then "present" end                                      
jq: error: Possibly unterminated 'if' statement at <top-level>, line 1:
.labels? | if length>0 then "present" end           
jq: 2 compile errors
```

**Comment by itchyny** [comment]
If you are using jq 1.6, please refer to https://stedolan.github.io/jq/manual/v1.6/.
In the master version, `else` is optional as implemented by #1825.
```sh
 % print '{"labels":{"owner1":"dls","archived":"","owner2":"data-services"}}' | jq -r '.labels? | if length>0 then "present" end'
present
 % jq --version
jq-master-a17dd32
```

**Comment by jeff-smith-pcln** [comment]
*Thanks for the reply.*

I'm using: *jq - commandline JSON processor [version 1.6]*

I simply downloaded the latest (1.6) from the
*https://stedolan.github.io/jq/download/
<https://stedolan.github.io/jq/download/>* page. If there's a newer version
which fixes the above issue, where would I be able to obtain it from?

*Sorry if I should know better, but there's a lot happening at work...*

*jeff smith*

**Comment by itchyny** [comment]
Current latest released version is 1.6. https://stedolan.github.io/jq/manual/ is the manual for `development version` and you can build yourself as described in README.md (https://github.com/stedolan/jq/blob/master/README.md).

**Comment by itchyny** [comment]
The document was fixed at #2271 (cc4efc49e1eedb98289347bf264c50c5c8656e7c) in May 2021, so closing the issue.
